back/app/services/permissions_service.rb

```diff
@@ -1,43 +1,16 @@
 # frozen_string_literal: true
 
 class PermissionsService
```

Comment:
Verify that permissions are still created automatically when creating a project or phase.

Reply:
Yes, permissions are created in engines/commercial/granular_permissions/app/services/granular_permissions/patches/side_fx_participation_context_service.rb